blocks/init/src/Blocks/components/button/components/button-options.js

```diff
@@ -4,17 +4,18 @@ import { Fragment } from '@wordpress/element';
 import { URLInput } from '@wordpress/block-editor';
 import { ColorPaletteCustom } from '@eightshift/frontend-libs/scripts/components';
 import { SelectControl, TextControl, Icon, ToggleControl } from '@wordpress/components';
-import { icons, getOptionColors } from '@eightshift/frontend-libs/scripts/editor';
+import { icons, getOptionColors, getOptions } from '@eightshift/frontend-libs/scripts/editor';
 import { checkAttr } from '@eightshift/frontend-libs/scripts/helpers';
 import manifest from './../manifest.json';
 
-const { options, title } = manifest;
+const { title } = manifest;
```

Comment:
If you're doing this outside of the component (since it's outside of `ButtonOptions`), doesn't this automatically make the variable global?

Reply:
no sure, we didn't have any problems on multiple block. Generally my understanding of JS is that there is no such thing as global unless you put it in a window object. Everything else is scoped by babel and webpack

Reply:
ehh, still it can be done couple of lines below inside the component where you do `const { options } = manifest;`, doesn't hurt

Reply:
this is just wild guessing. If it starts making problems I will move it

Reply:
I have asked JS guys

Reply:
I have this in all my components, didn't see any issues so far. Then again, I never had to override some parent options 🤔  What if this was causing issues for @kmaric69 ?

Reply:
Kristijan had different issues with his helper

Reply:
Just saying, there's no need to do it outside of the component (outside of `export const ButtonOptions = (attributes) => {`) and while it might not cause issues, it's a bit confusing. But if it's not causing issues I guess it's ok, will approve.